spec/requests/lend_process_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe "Lend Process", type: :request do
+  before do
+    @group = create(:group)
+    @owner = create(:user)
+    @item = create(:item, owning_user: @owner, groups_with_lend_permission: [@group])
+    @item.waitlist = create(:waitlist_with_item)
+    @borrower = create(:user)
+    @borrower.to_member_of!(@group)
+    @unauthorized = create(:user)
+    @waiting = create(:user)
+    @waiting.to_member_of!(@group)
+  end
+
+  describe "POST /request_lend/item" do
+
+    it "with permission" do
```

Comment:
It would be nice if the test's name would reflect the expected behavior, e.g. `it "with permission updates the lend status" do`.

Same thing for the other three tests in this file.